examples/kubernetes/snapshot/README.md

```diff
@@ -91,11 +91,12 @@ For information on creating an FSxZ snapshot see this [guide](https://docs.aws.a
     kubectl exec -ti fsx-restored-app -- tail -f /data/out.txt
     ```
 5. Delete all the created resources:
-   1. FSxZ filesystem:
+   1. FSx for OpenZFS file system:
    ```sh
-    kubectl delete -f restore-snapshot/,snapshot/,../dynamic-provisioning/manifests/,../dynamic-provisioning/filesystem/
+    kubectl delete -f restore-snapshot/,snapshot/,../dynamic-provisioning/pod/,../dynamic-provisioning/filesystem/
```

Comment:
We should maybe add a pod.yaml in restore-snapshot, or else step 4 can't occur. Also no need to delete the pod folder here as it is never applied.

Reply:
good catch, will do

Reply:
Haha, restore-snapshot already has a pod.yaml with the name fsx-restored-app, so we're fine there